layers.go

```diff
@@ -542,8 +542,8 @@ func (r *layerStore) Put(id string, parentLayer *Layer, names []string, mountLab
 			_, idInUse = r.byid[id]
 		}
 	}
-	if _, idInUse := r.byid[id]; idInUse {
-		return nil, -1, ErrDuplicateID
+	if l, idInUse := r.byid[id]; idInUse {
+		return l, -1, ErrDuplicateID
```

Comment:
The change is fine, but I'd much rather see 'layerId' or "layer" instead of 'l' here.  Personally I find it a lot clearer, easier to scan and I often have a hard time distinguishing between a lower case 'l' and a 1 (one).   Took me a couple of hard looks to determine what was what here between the two characters.

Reply:
Yes change the to layer and then we can merge.